azure/templates/post-deploy.yml

```diff
@@ -114,7 +114,7 @@ steps:
       then
         echo "Sandbox env tests"
         marker="not nhsd_apim_authorization and not sandbox_int and not mtls"
-      elif [[ $APIGEE_ENVIRONMENT == "int" ]]
+      elif [[ $APIGEE_ENVIRONMENT == "int" ]] || [[ $APIGEE_ENVIRONMENT == "ref" ]]
```

Comment:
I wouldn't think this can work. Won't this use the `int` credentials for `ref`, which are on different Apigee instances?

Reply:
I separate these two. I don't think this will cause anyway though. The int env only runs smoketests so I think those `INT_*` vars are unused. The credentials in the pipeline is the same for all variables.